dev/com.ibm.ws.logging_fat/fat/src/com/ibm/ws/logging/fat/ConsoleFormatTest.java

```diff
@@ -420,9 +475,10 @@ private void writeProperties(Properties props, FileOutputStream output) throws E
         }
     }
 
-    private static void setServerConfiguration(String consoleFormat, boolean useTraceSpec, boolean useIsoDateFormat, RemoteFile consoleLogFile) throws Exception {
+    private static void setServerConfiguration(LibertyServer Server, String consoleFormat, boolean useTraceSpec, boolean useIsoDateFormat,
```

Comment:
The variable name "Server" should be in lowercase, to meet java standards. Maybe, rename it to something else, so it doesn't conflict with "server"?